docs/source/contributing/linting.md

```diff
@@ -35,3 +30,60 @@ most importantly for .js/jsx files:
 - The editor should automatically flag unused imports
 - The editor should automatically (and properly) flag imported modules that are not found
 - The editor should provide automatic code formatting on save
+
+## Linting Volto core
+If you want to contribute to Volto core you need to perform several quality control tasks in order to ensure that you commits are not breaking the automated tests that Volto is performing on every commit.
+
+The first step in the testing hierarchy is the performing of code linting.
+
+Volto core should already perform these linting commands when commiting locally from the [husky](https://typicode.github.io/husky/get-started.html) integration, however if the the automated check doesn't happen when performing a commit or you want to get less information you can also run each linting task manually as you will learn how-to from the next section.
+
+If you want to see exactly which linting commands are set to run after a commit have a look at the [.lintstagedrc](https://github.com/plone/volto/blob/main/.lintstagedrc) file from the Volto project [main](https://github.com/plone/volto/) GitHub branch.
+
+Volto core performs linting using the following commands:
+- eslint - For finding problems in the project's code files
+- prettier - For uniform code formatting
+- stylelint - For uniform style formatting
+
+Although we can run the linting commands from the root of the project
+it will be easier to run the commands only for Volto core within the Volto core folder:
+
+  ```shell
+    cd packages/volto
```

Comment:
I think we should mention the `--filter @plone/volto` command from pnpm here (and maybe other places, whenever suitable).